### PR TITLE
Add updated API for queue

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by helfi92** [comment]
@eliperelman Just ran `yarn compile-clients`. Should I create a new tag after it gets merged? What about publishing it to npm.

**Comment by eliperelman** [comment]
For now I will take care of tags and releases, you should just worry about code updates.

**Comment by helfi92** [comment]
Perfect, thanks!

File: src/clients/Queue.js [hunk 1]
```diff
@@ -302,6 +313,12 @@ export default class Queue extends Client {
     return this.request(this.createArtifact.entry, args);
   }
 
+  // tbd
```
Comment (eliperelman) [inline src/clients/Queue.js]: Does someone want to update the schema comments before we publish this? cc: @djmitche

Comment (jhford) [reply src/clients/Queue.js]: That'd be me.  I'll write something up now and open a PR in the Queue.

Comment (jhford) [reply src/clients/Queue.js]: https://github.com/taskcluster/taskcluster-queue/pull/207

Comment (helfi92) [reply src/clients/Queue.js]: Awesome, thanks @jhford

Comment (helfi92) [reply src/clients/Queue.js]: Schema description updated.

**Comment by helfi92** [comment]
Just noticed that Travis is complaining of a linting issue. Will fix this.

**Comment by eliperelman** [comment]
Published in v3.1.0.

**Pull request merged**